lib/cli/extension-command.js

```diff
@@ -486,4 +487,49 @@ export default class ExtensionCommand {
     this.config.installedExtensions[ext].version = version;
     await this.config.write();
   }
-}
+
+  /**
+   * Runs a script cached inside the "scripts" field under "appium"
+   * inside of the driver/plugins "package.json" file
+   *
+   * @param {string} ext - name of the extension to run a script from
+   * @param {string} scriptName - name of the script to run
+   */
+  async run ({ext, scriptName}) {
+    if (!_.has(this.config.installedExtensions, ext)) {
+      const msg = `please install the ${this.type} first`;
```

Comment:
no need to define the string and throw in a separate line. just put the string literal in the Error constructor